lib/diagnostic.js

```diff
@@ -38,19 +54,36 @@ let owner = null;
 // Methods: vocalize
 
 // Replace null with your answer
-let pet = null;
+let pet = {
+  name: 'Bubbles';
+  species: 'Dog';
+  breed: 'Lab';
+  vocalization: function() {
+    this.vocalize
+  }
+  return vocalize;
+}
```

Comment:
Not quite. `vocalization` was supposed to be a property representing the noise that the pet makes (e.g. 'woof'), while `vocalize` was supposed to be a method that returned (or printed - the directions could be interpreted either way) the value of `vocalization`.